src/platform/site-wide/user-nav/sass/user-nav.scss

```diff
@@ -323,7 +323,9 @@ span.sidelines {
 #create-account a {
   border-top: 1px solid $color-gray-light;
   border-bottom: 1px solid $color-gray-light;
-  padding-bottom: 1.16em;
+  padding: 1.16em 0;
```

Comment:
Could this use the 32px design system utility `vads-u-padding-y--4` instead? cc @k80bowman

Reply:
Changing to `vads-u-padding-y--3` 🚀